java_gen/templates/custom/OFFlowDeleteStrictVer13.Builder_setActions.java

```diff
@@ -1,9 +1 @@
-
-    @Override
-    public OFFlowDeleteStrict.Builder setActions(List<OFAction> actions) throws UnsupportedOperationException {
-        OFInstructionApplyActionsVer13.Builder builder = new OFInstructionApplyActionsVer13.Builder();
-        builder.setActions(actions);
-        this.instructions = Collections.singletonList((OFInstruction)builder.build());
-        this.instructionsSet = true;
-        return this;
-    }
\ No newline at end of file
+//:: include("custom/OFFlowDeleteStrict.Builder_setActions.java", msg=msg, version=version, has_parent=False)
```

Comment:
That looks good. 

One other thought, since we're modifying that code anyway -- it seems having this helper method for `OFFlowDelete` and `OFFlowDeleteStrict` might not make all that much sense, because setting instructions isn't meaningful for deletes?